ocaml/gencert/test_data/pems/fail-06.pem

```diff
@@ -1,4 +1,74 @@
------BEGIN RSA PRIVATE KEY-----
```

Comment:
This is removing a failing unit-test of mismatched header and footer, could it be added back?